zubhub_frontend/zubhub/src/store/actions/activityActions.js

```diff
@@ -0,0 +1,21 @@
+import API from  '../../api/api'
+let ActivityAPI = new API()
+
+export const setActivities = activities => {
+    return dispatch => {
+      dispatch({
+        type: 'SET_ACTIVITIES',
```

Comment:
It’s better to use constants for action types, it avoids the typo issues in reducer/actions

Reply:
do you mean that i create those constants `const SET_ACTIVITIES='SET_ACTIVITIES'  ...` in the same file?

Reply:
Yes,
const SET_ACTIVITIES='SET_ACTIVITIES

But in different file since its used in both reducer and actions